addons/mail/models/mail_render_mixin.py

```diff
@@ -203,15 +215,22 @@ def _prepend_preview(self, html, preview):
         if preview:
             preview = preview.strip()
 
+        template_instructions = parse_inline_template(preview or '')
+        preview_markup = Markup('')
+        for string, expression in template_instructions:
+            if expression:
+                preview_markup += Markup('{}<t t-out="{}"/>').format(string, expression)
+            else:
+                preview_markup += escape(string)
```

Comment:
Maybe can we extract this code in a separate function (with a nice docstring) near `parse_inline_template` (`convert_inline_template_to_qweb` ?)

I would it would help to understand this code and will be easier to test